client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java

```diff
@@ -528,6 +530,10 @@ private void setHeaderParams(RequestBuilder request, Method method, Object[] arg
             return Mono.empty();
         }
 
+        if (String.class.equals(type) && !string.startsWith(QOUTES) && !"null".equalsIgnoreCase(string)) {
```

Comment:
qoutes -> quotes

Reply:
maybe im thinking wrong here. but should it not be: 

if( is String and starts with quote ) ?

Reply:
> qoutes -> quotes

I actually think it should be just quote, since it only contains one quotation mark  :)

Reply:
If the string starts with quotes we need to run it through the objectMapper

Reply:
I will change spelling and remove an S

The null test is already implemented in RestClientFactoryTest